compiler/qsc_passes/src/entry_point.rs

```diff
@@ -139,6 +145,9 @@ impl<'a> Visitor<'a> for EntryPointFinder<'a> {
             if item.attrs.iter().any(|a| a == &Attr::EntryPoint) {
                 self.callables.push((callable, item.id));
             }
+            if callable.name.name.as_ref().to_lowercase() == "main" {
```

Comment:
Why the 'to_lowercase'? Is 'mAiN' valid (Or even just `main`)?

Reply:
Responded in other comment. I'll remove the case insensitivity.